drivers/usb/device/Kconfig

```diff
@@ -21,6 +21,36 @@ config USB_DW_IRQ_PRI
 	help
 	USB interrupt priority.
 
+config USB_DC_STM32
+	bool
+	prompt "USB device controller driver for STM32 devices"
+	depends on HAS_STM32CUBE
+	depends on SOC_SERIES_STM32F4X
+	default n
+	help
+	STM32 family USB device controller Driver.
+
+config USB_DC_STM32_IRQ_PRI
+	int "USB STM32 Driver Interrupt priority"
+	depends on USB_DC_STM32
+	default 3
+	help
+	USB interrupt priority.
+
+config USB_DC_STM32_EP_NUM
+	int
+	depends on USB_DC_STM32
+	default 4
+	help
+	Number of supported endpoints (including ep0).
+
+config USB_DC_STM32_RAM_SIZE
+	int
+	depends on USB_DC_STM32
+	default 1280
```

Comment:
The minimum RAM SIZE is 512 (for device with PMA). Could we use this value as default.

Reply:
done